### PR TITLE
Update PR-powershell-preview.yml

### PR DESCRIPTION
**Pull request by TravisEz13**
# PR Summary

### CODE REVIEW

File: .github/workflows/PR-powershell-lts.yml [hunk 1]
```diff
@@ -36,8 +33,18 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - uses: dorny/paths-filter@v2
```
Comment (TravisEz13) [inline .github/workflows/PR-powershell-lts.yml:36]:
```suggestion
      - uses: dorny/paths-filter@4512585405083f25c027a35db413c2b3b9006d50
```

File: .github/workflows/PR-powershell-preview.yml [hunk 2]
```diff
@@ -35,9 +32,19 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+        
+      - uses: dorny/paths-filter@v2
```
Comment (TravisEz13) [inline .github/workflows/PR-powershell-preview.yml:36]:
```suggestion
      - uses: dorny/paths-filter@4512585405083f25c027a35db413c2b3b9006d50
```

File: .github/workflows/PR-powershell.yml [hunk 3]
```diff
@@ -38,10 +35,23 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - uses: dorny/paths-filter@v2
```
Comment (TravisEz13) [inline .github/workflows/PR-powershell.yml:38]:
```suggestion
      - uses: dorny/paths-filter@4512585405083f25c027a35db413c2b3b9006d50
```

File: .github/workflows/PR-powershell.yml [hunk 4]
```diff
@@ -38,10 +35,23 @@ jobs:
...
+        id: filter
+        with:
+          filters: |
+            workflows:
+              - '.github/workflows/PR-powershell.yml'
+              - 'Formula/powershell.rb'
+
+      - name: Prepare Agent
+        if: steps.filter.outputs.workflows == 'true'
+
```
Comment (TravisEz13) [inline .github/workflows/PR-powershell.yml:48]:
```suggestion
```

File: .github/workflows/PR-powershell.yml [hunk 5]
```diff
@@ -38,10 +35,23 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - uses: dorny/paths-filter@4512585405083f25c027a35db413c2b3b9006d50
```
Comment (TravisEz13) [inline .github/workflows/PR-powershell.yml:38]:
```suggestion
      - name: Check for applicable files
         uses: dorny/paths-filter@4512585405083f25c027a35db413c2b3b9006d50
```

File: .github/workflows/PR-powershell-preview.yml [hunk 6]
```diff
@@ -35,9 +32,19 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+        
+      - uses: dorny/paths-filter@4512585405083f25c027a35db413c2b3b9006d50
```
Comment (TravisEz13) [inline .github/workflows/PR-powershell-preview.yml:36]:
```suggestion
      - name: Check for applicable files
         uses: dorny/paths-filter@4512585405083f25c027a35db413c2b3b9006d50
```

File: .github/workflows/PR-powershell-lts.yml [hunk 7]
```diff
@@ -36,8 +33,18 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - uses: dorny/paths-filter@4512585405083f25c027a35db413c2b3b9006d50
```
Comment (TravisEz13) [inline .github/workflows/PR-powershell-lts.yml:36]:
```suggestion
      - name: Check for applicable files
         uses: dorny/paths-filter@4512585405083f25c027a35db413c2b3b9006d50
```

File: .github/workflows/PR-powershell-lts.yml [hunk 8]
```diff
@@ -36,8 +33,19 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Check for applicable files
+         uses: dorny/paths-filter@4512585405083f25c027a35db413c2b3b9006d50
```
Comment (adityapatwardhan) [inline .github/workflows/PR-powershell-lts.yml:37]: indent

File: .github/workflows/PR-powershell-preview.yml [hunk 9]
```diff
@@ -35,9 +32,20 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+        
+      - name: Check for applicable files
+         uses: dorny/paths-filter@4512585405083f25c027a35db413c2b3b9006d50
```
Comment (adityapatwardhan) [inline .github/workflows/PR-powershell-preview.yml:37]: same here

File: .github/workflows/PR-powershell.yml [hunk 10]
```diff
@@ -38,10 +35,21 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Check for applicable files
+         uses: dorny/paths-filter@4512585405083f25c027a35db413c2b3b9006d50
```
Comment (adityapatwardhan) [inline .github/workflows/PR-powershell.yml:39]: indent

File: .github/workflows/PR-powershell-daily.yml [hunk 11]
```diff
@@ -37,9 +34,20 @@ jobs:
...
         uses: actions/checkout@v2
+        
+      - name: Check for applicable files
+        uses: dorny/paths-filter@4512585405083f25c027a35db413c2b3b9006d50
+        id: filter
+        with:
+          filters: |
+            workflows:
+              - .github/workflows/PR-powershell-preview.yml
+              - Formula/powershell-preview.rb
```
Comment (TravisEz13) [inline .github/workflows/PR-powershell-daily.yml:45]:
```suggestion
              - .github/workflows/PR-powershell-daily.yml
              - Formula/powershell-daily.rb
```

File: .github/workflows/PR-powershell-lts.yml [hunk 12]
```diff
@@ -36,8 +33,19 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Check for applicable files
+        uses: dorny/paths-filter@4512585405083f25c027a35db413c2b3b9006d50
+        id: filter
+        with:
+          filters: |
+            workflows:
+              - '.github/workflows/PR-powershell-preview.yml'
```
Comment (adityapatwardhan) [inline .github/workflows/PR-powershell-lts.yml:42]:
```suggestion
              - '.github/workflows/PR-powershell-lts.yml'
```

File: .github/workflows/PR-powershell-lts.yml [hunk 13]
```diff
@@ -36,8 +33,19 @@ jobs:
...
         uses: actions/checkout@v2
 
+      - name: Check for applicable files
+        uses: dorny/paths-filter@4512585405083f25c027a35db413c2b3b9006d50
+        id: filter
+        with:
+          filters: |
+            workflows:
+              - '.github/workflows/PR-powershell-lts.yml'
+              - 'Formula/powershell-preview.rb'
```
Comment (TravisEz13) [inline .github/workflows/PR-powershell-lts.yml:43]:
```suggestion
              - 'Formula/powershell-lts.rb'
```

**Review by adityapatwardhan — APPROVED** [review]

**Review by adityapatwardhan — APPROVED** [review]

**Pull request merged**